tmt/steps/prepare/install.py

```diff
@@ -522,7 +521,7 @@ class PrepareInstallData(tmt.steps.prepare.PrepareStepData):
         )
 
 
-@tmt.steps.provides_method('install')
+@ tmt.steps.provides_method('install')
```

Comment:
Was this change intentional?

Reply:
Oops, nope, that does not look like an intention. It shouldn't have any effect, but it's not supposed to happen.

Reply:
Ok, we can fix with some other patch.